examples/geojson-animation/webpack.config.js

```diff
@@ -5,6 +5,16 @@
 const resolve = require('path').resolve;
 const webpack = require('webpack');
 
+const BABEL_CONFIG = {
```

Comment:
Why is this needed inside an example? I thought this was only needed when one directory referred to another outside its tree. The `examples` folder is intended to contain stand-alone examples.